test/k8s-integration/main.go

```diff
@@ -34,11 +34,12 @@ var (
 	pkgDir = flag.String("pkg-dir", "", "the package directory")
 
 	// Cluster configs.
-	doNetworkSetup = flag.Bool("do-network-setup", true, "whether to setup and then cleanup a VPC network during the test")
-	bringupCluster = flag.Bool("bringup-cluster", true, "bringup a GKE cluster before the e2e test")
-	testVersion    = flag.String("test-version", "", "version of k8s binary to download and use for the e2e test")
-	numNodes       = flag.Int("num-nodes", -1, "the number of nodes in the test cluster")
-	imageType      = flag.String("image-type", "cos_containerd", "the node image type to use for the cluster")
+	doNetworkSetup  = flag.Bool("do-network-setup", true, "whether to setup and then cleanup a VPC network during the test")
+	doMultiNICSetup = flag.Bool("multi-nic-setup", true, "whether to setup multi nic on cluster or not")
+	bringupCluster  = flag.Bool("bringup-cluster", true, "bringup a GKE cluster before the e2e test")
+	testVersion     = flag.String("test-version", "", "version of k8s binary to download and use for the e2e test")
+	numNodes        = flag.Int("num-nodes", -1, "the number of nodes in the test cluster")
```

Comment:
We currently set the num of nodes for default node pool to 3. https://github.com/GoogleCloudPlatform/lustre-csi-driver/blob/main/test/run-k8s-integration.sh#L50

Should we introduce a new flag to control the size of the multi-NIC pool? This would allow us to optimize resource usage (e.g., 1 node for default, 2-3 for multi-NIC).